<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupItem</class>
 <widget class="QWidget" name="GroupItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>70</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>260</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
background-color:rgb(84, 92, 255);
border: 2px solid gray;
margin-top: 0px;
 }</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_pic">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>4</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/resource/image/talkwindow/resource/image/talkwindow/group.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_info">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>29</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#80129f;&quot;&gt;群的宗旨：打败瓜不甜主义&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_num">
    <property name="geometry">
     <rect>
      <x>76</x>
      <y>47</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	border-color: rgb(255, 250, 5);
    font: 100 12pt &quot;微软雅黑&quot;;
    color: rgb(255, 250, 5);
    background-color: rgba(255, 193, 245, 0%);
}
QLabel:hover{
	border-color: rgb(0, 170, 255);
	color: rgb(255,125, 0);
	font: 100 20pt &quot;微软雅黑&quot;;
    background-color: rgba(255, 193, 245, 0%);
}</string>
    </property>
    <property name="text">
     <string>群人数：100人</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_name">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>2</y>
      <width>187</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	border-color: rgb(23, 30, 5);
    font: 100 12pt &quot;微软雅黑&quot;;
    color: rgb(33, 250, 127);
    background-color: rgba(255, 193, 245, 0%);
}
QLabel:hover{
	border-color: rgb(0, 170, 255);
 
	color: rgb(255, 0, 0);
	font: 100 20pt &quot;微软雅黑&quot;;
    background-color: rgba(255, 193, 245, 0%);
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;瓜瓜组&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
